translate/storage/test_mozilla_lang.py

```diff
@@ -65,7 +65,7 @@ def test_active_flag(self):
         """Test the ## active ## flag"""
         lang = ("## active ##\n"
```

Comment:
In all examples I have checked the file metadata is separated by two empty lines (like the strings), so here there are two missing newlines. It is kinda unrelated to the PR, but it would be great to handle.